### PR TITLE
Civ 1241 send financial details

### PR DESCRIPTION
**Pull request by osheaf01**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-1241

### Change description ###
Send your financial details

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by sampankumar — APPROVED** [review]

File: src/main/common/utils/urlFormatter.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export function getBaseUrlWithIdParam(id: string): string {
```
Comment (laarmada) [inline src/main/common/utils/urlFormatter.ts:1]: we should replace that for constructResponseUrlWithIdParams .. integrated in master + url constant file?

**Review by laarmada — COMMENTED** [review]
good testing job!

**Review by LeonardoPalmeiro — APPROVED** [review]

File: src/main/routes/urls.ts [hunk 2]
```diff
@@ -25,3 +25,8 @@ export const CITIZEN_PARTNER_PENSION_URL = `${STATEMENT_OF_MEANS_URL}/partner/pa
 export const CITIZEN_PARTNER_DISABILITY_URL = `${STATEMENT_OF_MEANS_URL}/partner/partner-disability`;
 export const CITIZEN_DEPENDANTS_URL = `${STATEMENT_OF_MEANS_URL}/dependants`;
 export const CITIZEN_BANK_ACCOUNT_URL = `${STATEMENT_OF_MEANS_URL}/bank-accounts`;
+export const FINANCIAL_DETAILS_URL  = `${BASE_CASE_RESPONSE_URL}/financial-details`;
```
Comment (mounikahmcts) [inline src/main/routes/urls.ts:28]: This url should be removed.

Comment (TatsyMaslova) [reply src/main/routes/urls.ts]: done

**Pull request merged**